lib/matplotlib/__init__.py

```diff
@@ -283,6 +283,12 @@ def wrapper():
 _ExecInfo = namedtuple("_ExecInfo", "executable version")
 
 
+class ExecutableNotFoundError(FileNotFoundError):
+    """Error raised when an executable that Matplotlib optionally
+    depends on can't be found."""
+    pass
```

Comment:
You don't need the "pass" here.

Also the docstring would typically be formatted
```
"""
Error raised...
"""
```
(basically either it fits in one line and we write
```
"""A one line docstring."""
```
or it doesn't and we write
```
"""
A multiline
docstring.
"""
```
or even
```
"""
One line, but just too long to have opening and closing quotes on the same line.
"""
```